src/server/common/helpers/auth/user-session.test.js

```diff
@@ -122,7 +126,7 @@ describe('#userSession', () => {
         scope: ['aabe63e7-87ef-4beb-a596-c810631fc474', 'admin', 'tenant'],
         uuidScope: ['aabe63e7-87ef-4beb-a596-c810631fc474'],
         expiresIn: 3600000,
-        expiresAt: new Date('2025-02-28T01:00:00.000Z')
+        expiresAt: expect.any(Date)
```

Comment:
FYI the jest.setSystemTime sets the time for the code in this test. If you wanted to expect an exact date here you can